apps/web/src/views/Notifications/components/NotificationItem/NotificationItem.tsx

```diff
@@ -0,0 +1,229 @@
+    return (
+      <FlexGap borderRadius={16} backgroundColor="tertiary" paddingY="2px" paddingX="6px" alignItems="center" gap="2px">
+        <ArrowDropDownIcon color="text" />
+        <Text fontSize="12px">{`${isAPR ? 'APR' : ''} ${hasFallen ? 'Down' : 'Up'} ${percentageChange}%`}</Text>
+      </FlexGap>
+    )
+  }
+  return <></>
+}
+
+const formatStringWithNewlines = (inputString: string) => {
+  const cleanStr = removeTokensFromAPRString(inputString)
+  return cleanStr.split('\n').map((line: string, index: number) => (
+    // eslint-disable-next-line react/no-array-index-key
+    <Text key={`message-line-${index}`} lineHeight="20px" fontWeight={400} color="textSubtle">
+      {line}
+    </Text>
+  ))
+}
+
+const NotificationItem = ({ title, description, date, image, url }: INotificationprops) => {
+  const [show, setShow] = useState<boolean>(false)
+  const [elementHeight, setElementHeight] = useState<number>(0)
+  const formattedDate = formatTime(Math.floor(date / 1000).toString())
+  const containerRef = useRef(null)
+  const contentRef = useRef<HTMLDivElement>(null)
+  const { t } = useTranslation()
+  const formatedDescription = formatStringWithNewlines(description)
+
+  const handleExpandClick = useCallback(
+    (e: React.MouseEvent<HTMLDivElement>) => {
+      const target = e.target as HTMLDivElement
+      if (target.tagName !== 'BUTTON') setShow(!show)
+    },
+    [show],
+  )
+  // eslint-disable-next-line @typescript-eslint/no-shadow
+  const getLinkText = (title: string) => {
+    if (title.includes('APR Update')) return t('View Farm')
+    if (title.includes('Balance')) return t('Buy Crypto')
+    return t('View Link')
+  }
+
+  useEffect(() => {
+    if (contentRef.current) setElementHeight(contentRef.current.scrollHeight)
+  }, [])
+
+  return (
+    <StyledNotificationWrapper ref={containerRef} onClick={handleExpandClick}>
+      <ContentsContainer>
+        <Flex flexDirection="column" width="100%">
+          <Flex justifyContent="space-between" width="100%">
+            <NotificationImage image={image} title={title} message={description} />
+            <Flex flexDirection="column" width="100%">
+              <Text fontWeight={600}>{title}</Text>
+              <FlexGap alignItems="center" gap="6px">
+                <Dot show color="success" />
+                <Text fontSize="13px" color="textSubtle">
+                  {formattedDate}
+                </Text>
+                <NotificationBadge title={title} message={description} />
+              </FlexGap>
+            </Flex>
+            <BottomRow show={show} elementHeight={elementHeight} />
+          </Flex>
+          <Description ref={contentRef} show={show} elementHeight={elementHeight}>
+            <Text>{formatedDescription}</Text>
+            {url !== '' ? (
+              <StyledLink hidden href={url} target="_blank" rel="noreferrer noopener">
+                {getLinkText(title)}
+              </StyledLink>
+            ) : null}
```

Comment:
How about change it to
`{url && (<StyledLink>)</StyledLink> }`

Reply:
i always used {condition ? truecase : null} instead of {condition && truecase} i remeber reading somewhere its better to do this over the former. what do you think, not really sure if it makes that much of a diff, and i do get that the && syntax looks nicer.... fits on one line

Reply:
We can change it to `&&`